pt-BR/mail/chrome/messenger/subscribe.properties

```diff
@@ -35,9 +35,9 @@
 #
 # ***** END LICENSE BLOCK *****
 
-subscribeLabel-nntp=Inscrever os newsgroups selecionados:
+subscribeLabel-nntp=Inscrever os grupos de notícias selecionados:
 subscribeLabel-imap=Inscrever as pastas selecionadas:
```

Comment:
CORRECTIONS NEEDED it should be subscribeLabel-imap=Me inscrever nas pastas selecionadas: